grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -3919,6 +3921,60 @@ def section_network_connection(outer_panels):
     "type": "query",
 }
 
+stream_graph_extra_json = {
+    "options": {
+        "useBackground": True,
+        "nodeSize": {"minWidth": 30, "minHeight": 30},
+        "legend": {
+            "show": False,
+            "stats": ["mean", "last", "min", "max", "sum"],
+        },
+        "contentUrl": "http://localhost:5691/api/flowchart/table/${table}",
```

Comment:
Cool! In this way the Grafana can display a dynamic graph, generated by the Rust code

Reply:
Yeah, but I find limitations. 😕 The requests seem to be sent from the browser, so the connectivity might be a problem.

Reply:
The JSON data source might help to fetch the flowchart content, which supports sending requests from the server:

https://github.com/simPod/GrafanaJsonDatasource